lib/trento/clusters.ex

```diff
@@ -148,16 +148,9 @@ defmodule Trento.Clusters do
     |> select_merge([c, e], %{cib_last_written: e.cib_last_written})
   end
 
-  defp get_filesystem_type(cluster_id) do
-    query =
-      from(c in ClusterReadModel,
-        where:
-          c.id == ^cluster_id and c.type == ^ClusterType.ascs_ers() and is_nil(c.deregistered_at)
-      )
-
+  defp get_filesystem_type(%ClusterReadModel{} = cluster) do
```

Comment:
nitpick: Could we move this `get_filesystem_type` function after `maybe_request_checks_execution`? To follow the order precedence

Reply:
Here we go 696a3cc728fe87092d5981877a49c51b647bf21b